### ISSUE TITLE
Nudge emails

### ISSUE THREAD

**Issue by aergonaut** [aergonaut/cody#320]
Similar to the new reviews summary, we could send an "aging" reviews summary, to show reviewers what code reviews they have which have been open for a while. This could indicate code reviews that are lagging or stalled and that the reviewer should go try to unblock.

**Comment by deepakmahakale** [comment]
Hey, 
any updates on this one? 
Actually, we were looking for a config to send emails to reviewers if it's been assigned to them for a few days but not addressed.

**Comment by aergonaut** [comment]
I have not had time to consider this idea recently. It is around the middle of my list at the moment. Please feel free to make a PR if you are interested in adding it!

**Comment by aergonaut** [comment]
Won't do. Teams wanting automated reminders for reviewers should encourage users to use GitHub's built-in [reminders](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/managing-your-scheduled-reminders).
